Runtime/Scripts/Export/ExportJobs.cs

```diff
@@ -85,6 +85,98 @@ public unsafe struct ConvertPositionFloatJob : IJobParallelFor {
             }
         }
         
+        [BurstCompile]
+        public unsafe struct ConvertMatrixJob : IJobParallelFor {
+
+            public uint byteStride;
+            
+            [ReadOnly]
+            [NativeDisableUnsafePtrRestriction]
+            public NativeArray<float4x4> input;
+            
+            [WriteOnly]
+            [NativeDisableUnsafePtrRestriction]
+            public NativeArray<float4x4> output;

```

Comment:
Optimization suggestion:
Most export IJob* have separate input and result/output arrays, since the input usually is the source vertex stream of the mesh.
In this case the input is not given, but constructed. We could convert the matrices in-place and safe memory by not using a dedicated results array.